<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDialogCodeDialog</class>
 <widget class="QDialog" name="CreateDialogCodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forms code generator</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Classname:</string>
       </property>
       <property name="buddy">
        <cstring>classNameCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="classNameCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the class that should get the forms code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="newButton">
       <property name="toolTip">
        <string>Press to generate a new forms class</string>
       </property>
       <property name="text">
        <string>&amp;New...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Filename:</string>
       </property>
       <property name="buddy">
        <cstring>filenameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="filenameEdit">
       <property name="toolTip">
        <string>Displays the name of the file containing the code</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filter &amp;with:</string>
       </property>
       <property name="buddy">
        <cstring>filterEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="E5ClearableLineEdit" name="filterEdit">
       <property name="toolTip">
        <string>Enter a regular expression to filter the list below</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="slotsView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>E5Gui/E5LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>classNameCombo</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>filenameEdit</tabstop>
  <tabstop>filterEdit</tabstop>
  <tabstop>slotsView</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateDialogCodeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
